backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/UpdateVmCommand.java

```diff
@@ -957,6 +958,14 @@ private void initNuma() {
             getVm().setvNumaNodeList(vNumaNodeList);
         }
 
+        if (getParameters().getVm().getCpuPinningPolicy() == CpuPinningPolicy.RESIZE_AND_PIN_NUMA) {
+            getParameters().setUpdateNuma(false);
+        }
+
+        if (getParameters().isUpdateNuma() == null) {
```

Comment:
Since now it's Boolean, I think it will also required to have null check on `updateVmNumaNodes()`

Reply:
As we always call `initNuma` first I do not think it is necessary as we set the value there
```
getParameters().setUpdateNuma(!vNumaNodeList.equals(getParameters().getVm().getvNumaNodeList()));
```